src/services/database/migrate.ts

```diff
@@ -1,16 +1,23 @@
 import { config } from "dotenv";
 config({ path: ".env.local" });
 import { migrate as migrateVercel } from "drizzle-orm/vercel-postgres/migrator";
-import { migrate as migrateNode } from "drizzle-orm/node-postgres/migrator";
+import { migrate as migrateNode } from "drizzle-orm/postgres-js/migrator";
```

Comment:
Please use the same naming convention. As you already named it `drizzlePostgres` in `src/services/database/migrate.ts`